client/lib/create-selector/index.js

```diff
@@ -75,6 +79,10 @@ export default function createSelector( selector, getDependants = DEFAULT_GET_DE
 	let lastDependants;
 
 	return Object.assign( function( state, ...args ) {
+		if ( Array.isArray( getDependants ) ) {
+			getDependants = makeSelectorFromArray( getDependants );
+		}
+
```

Comment:
>Do you think it's a worthwhile addition to `createSelector`?

Yeah, I think it'd be a reasonable option to add as a convenience for "if any of these selectors changes, bust the cache". Should we move this up to the parent scope? I was going to say "so we don't recompute it every time" but I suppose since we're reassigning the original variable that wouldn't be the case, and you could even make the argument that the deferred computation of the `getDependants` selector could be a nice thing. I'll leave it to you to choose.

We should add some tests, and a doc block would be nice.